### PR TITLE
added gnu-cobol free option

### PR DESCRIPTION
**Pull request by krzkaczor**
Added option to use 'free' mode of gnu cobol. It is super useful IMO - you don't have to care about indents etc.

### CODE REVIEW

File: lib/compile/index.js [hunk 1]
```diff
@@ -41,10 +41,16 @@ function Compile(input, options, callback) {
         if (!exists) {
             return callback(new Error("Couldn't find the cobc executable in the PATH. Make sure you installed Open Cobol."));
         }
-        Exec(OArgv({
+
+        var args = {
             x: true
-          , _: input
-        }, "cobc"), {
+            , _: input
```
Comment (IonicaBizau) [inline lib/compile/index.js]: Move this line with two spaces on the left. :smile:

**Comment by IonicaBizau** [comment]
Niiiice, @krzkaczor! :sparkles: Add yourself as contributor and it will be ready for merge. :ship:

**Comment by krzkaczor** [comment]
Done!

File: README.md [hunk 2]
```diff
@@ -138,6 +138,7 @@ Runs COBOL code from Node.JS side.
 - **Function|String|Path** `input`: A function containing a comment with inline COBOL code, the cobol code itself or a path to a COBOL file.
 - **Object** `options`: An object containing the following fields:
  - `cwd` (String): Where the COBOL code will run (by default in the current working directory)
+ - `free` (Boolean): Use free option while compiling with GnuCobol
```
Comment (IonicaBizau) [inline README.md]: Oh, and [add this line in the code too](https://github.com/IonicaBizau/node-cobol/blob/fbbe2d25cbd8ec7c7435fab9053ce5943fe577e0/lib/index.js#L23). Actually the README.md file is generated via Blah. I wrote on my blog [how I use Blah to generate docs](http://ionicabizau.net/blog/25-increasing-productivity-using-blah-2-0-0). :star2:

**Comment by krzkaczor** [comment]
Good point, didn't know about it. Fix'd

**Comment by IonicaBizau** [comment]
:ship: `1.3.0` :tada: Thanks! :apple:

**Pull request merged**